includes/Rest/AnyCommentRestComment.php

```diff
@@ -627,7 +627,7 @@ public function create_item ( $request ) {
 		if ( is_wp_error( $check_comment_lengths ) ) {
 			$error_code = $check_comment_lengths->get_error_code();
 
-			return new WP_Error( $error_code, __( 'Comment field exceeds maximum length allowed.', 'anycomment' ), array( 'status' => 400 ) );
+			return new WP_Error( $error_code, __( 'Your comment is too lengthy.', 'anycomment' ), array( 'status' => 400 ) );
```

Comment:
Maybe `too long` would be better? :eyes:

Reply:
Sure:

```suggestion
			return new WP_Error( $error_code, __( 'Your comment is too long.', 'anycomment' ), array( 'status' => 400 ) );
```